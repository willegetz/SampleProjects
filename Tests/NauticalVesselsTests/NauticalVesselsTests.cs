using ApprovalTests;
using ApprovalTests.Reporters;
using ShipYard;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NauticalVesselsTests
{
    [TestClass]
    [UseReporter(typeof(VisualStudioReporter))]
    public class NauticalVesselsTests
    {
        [TestMethod]
        public void TestGetABoat()
        {
            // arrange
            // Instantiate the ship factory
            var shipyard = new Construction();
            // act
            // Get a clipper ship
            var clipperShip = shipyard.GetClipper();
            // assert
            // Verify you have a clipper ship
            Approvals.Verify(clipperShip);
        }
    }
}
