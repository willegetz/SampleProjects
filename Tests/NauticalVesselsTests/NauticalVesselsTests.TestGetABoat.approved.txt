This is a 3 masted, Clipper Ship ship with 200 MO of cargo space.